nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/security/ConanContentValidator.java

```diff
@@ -67,6 +67,8 @@ else if (contentName.endsWith("/conanfile.py")) {
       }
       else if (ConanFacetUtils.isPackageSnapshot(contentName)) {
         return ContentTypes.APPLICATION_JSON;
+      } else if (ConanFacetUtils.isDigest(contentName)) {
+        return ContentTypes.APPLICATION_JSON;
       }
```

Comment:
pls. use one style for else if. new str will be useful here

Reply:
fixed fixed https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/97/commits/d7566f3ca7a5297d9e90aac735efefebc064b31d